src/shopkit/index.js

```diff
@@ -1,10 +1,24 @@
 import React, { createContext } from 'react'
-import { createClient } from '@moltin/request'
+import { MoltinClient } from '@moltin/request'
 
 import { CartProvider, Cartkit } from './Cartkit'
 import { CustomerProvider, Customerkit } from './Customerkit'
 import { CheckoutProvider, Checkoutkit } from './Checkoutkit'
 
+class MoltinNodeStorageAdapter {
```

Comment:
@ynnoj can we rename this to `MoltinLocalStorageAdapter` instead?

Reply:
@notrab Forgot to commit that change. Probably worth abstracting to a separate package as noted in commit message.